numpy/random/_generator.pyx

```diff
@@ -3627,6 +3627,12 @@ cdef class Generator:
         nonnegative-definite). Otherwise, the behavior of this method is
         undefined and backwards compatibility is not guaranteed.
 
+        This function internally uses linear algebra routines, and thus results
+        may not be identical (even up to precision) across architectures, OSes,
+        or even builds. For example, this is likely if `cov` has multiple equal
+        singular values and `method` is `'svd'` (default). In this case,
+        `method = 'cholesky'` may be more robust.
```

Comment:
```suggestion
        ``method='cholesky'`` may be more robust.
```
We need double ticks almost exclusively (for parameter names either way is OK, because numpydoc says single ticks even if it ends up with sphinx warnings.  So at least `'svd'` also should get it, I believe.

As Melissa noted, the examples section is missing the ticks/math mode in the text.  I don't think we need to add it here.  I suspect only "Examples" are simply run and that plotting is in the notes.  That seems OK (and unrelated)?

I am not worried about this breaking CI, but the skip CI means I can't link the doc build to show how it currently renders.

Reply:
I added ticks everywhere.